packages/image-utils/src/Image.ts

```diff
@@ -35,6 +35,9 @@ async function resizeAsync(imageOptions: ImageOptions): Promise<Buffer> {
     if (imageOptions.removeTransparency) {
       jimp.colorType(2);
     }
+    if (imageOptions.circle) {
+      jimp.circle();
```

Comment:
would like to do something similar here to how we're doing this same operation with `sharp`, that way this could be a setting to round the corners instead of just cut out a circle, but was running into some mimeType issues trying to create a `Sharp` image. but can look into that soon

Reply:
perhaps a borderRadius property would scale better

Reply:
changed it to `borderRadius: number`. Right now if using Jimp any number will just make the image a circle though